Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -1,213 +0,0 @@
-# Getting started with MRTK
```

Comment:
@hferrone, I thought we agreed we would keep the old Getting StartedWithMRTK.md and have it say that the page has been replaced by installation.md